cacher/redis/redis_test.go

```diff
@@ -0,0 +1,108 @@
+package redis
+
+import (
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/cryptopay-dev/yaga/cacher"
+	"github.com/go-redis/redis"
+	"github.com/stretchr/testify/assert"
+)
+
+const keyTpl = "test:key:%d"
+
+func closedCacher() cacher.Cacher {
+	c := redis.NewClient(&redis.Options{
+		Addr: os.Getenv("TEST_REDIS_ADDR"),
+	})
+	defer c.Close()
+	return FromConnection(c)
+}
+
+func defaultCacher() cacher.Cacher {
+	return New(
+		Address(os.Getenv("TEST_REDIS_ADDR")),
+		DB(0),
+		Password(""),
+	)
+}
+
+func TestService(t *testing.T) {
+	c := defaultCacher()
+
+	var (
+		i    = 1
+		val  int
+		key  = fmt.Sprintf(keyTpl, 0)
+		ttl1 = time.Second * time.Duration(100)
```

Comment:
mb
```ttl1 = time.Second * time.Duration(100)```
=>
```ttl1 = time.Second * 100```

https://play.golang.org/p/AGeMCq9efTz

Reply:
yep, it's make sense